Materialy/TestowanieBazDanych/TestowanieBazDanychNarzedzia.md

```diff
@@ -0,0 +1,14 @@
+[![](https://img.shields.io/badge/Facebook-%23TestowanieOprogramowania-blue.svg)](https://www.facebook.com/groups/TestowanieOprogramowania/)
+
+
+# Testowanie baz danych
```

Comment:
Będzie to plik z informacjami o testowaniu baz danych jak sugeruje ta linijka, czy jednak wpis zawierający tylko otwartoźródłowe narzędzia do testowania baz danych?